sdk/src/auth/index.ts

```diff
@@ -347,7 +347,7 @@ class AWF_Auth {
       return Promise.resolve(null);
     }
     const localUser = localStorage.getItem(this.currentUserKey);
-    if (localUser) {
+    if (localUser && !this._lockSignIn) {
```

Comment:
The lock happens after this block, it shouldn't have any effect this change.

Reply:
Actually this would work only for multiple `getCurrentUser()` requests in paralel

Reply:
On some refresh,  automatic sign in doesn't work and the login placeholder keeps on loading. It seems that the automatic sign in triggered via calling `getCurrentUser` never emits SIG_IN event.  It does appear that the  `eth_requestAccounts` request to MetaMask is never fulfilled and the lines after this request never executes. 

It has been quite tough to reproduce the issue being observed on the PR review locally and the best thing now is add some console logs and debug from the PR previews.